tests/std/tests/native_17_matrix.lst

```diff
@@ -9,10 +9,12 @@ PM_CL="/EHsc /MD /D_ITERATOR_DEBUG_LEVEL=0 /std:c++latest"
 PM_CL="/EHsc /MD /D_ITERATOR_DEBUG_LEVEL=0 /std:c++17"
 PM_CL="/EHsc /MD /D_ITERATOR_DEBUG_LEVEL=1 /std:c++latest"
 PM_CL="/EHsc /MD /D_ITERATOR_DEBUG_LEVEL=0 /std:c++latest /Zc:char8_t-"
-PM_CL="/EHsc /MDd /D_ITERATOR_DEBUG_LEVEL=0 /std:c++latest /Zc:wchar_t-"
+# C5105: macro expansion producing 'defined' has undefined behavior is emitted by windows.h
+PM_CL="/EHsc /MDd /D_ITERATOR_DEBUG_LEVEL=0 /std:c++latest /Zc:wchar_t- /Zc:preprocessor /wd5105"

```

Comment:
I observe that the `impure_matrix` and `locale0_implib_cpp_matrix` are changing `/MD /D_ITERATOR_DEBUG_LEVEL=0` (i.e. default release mode), but this `native_17_matrix` is changing `/MDd /D_ITERATOR_DEBUG_LEVEL=0` (i.e. non-default debug mode). Is there a reason for this difference? Otherwise, I think the `impure_matrix`'s pattern is better. Occurs in the other files.

Reply:
I recall there being a reason for this, but I don't remember what it was

Reply:
ah the reason is that for the matrices that contain /permissive- we want to test that and it's on the debug/IDL0 configuration. I'll keep it there in those matrices.